.github/workflows/main.yml

```diff
@@ -329,9 +342,11 @@ jobs:
         run: pnpm cypress install && pnpm cypress cache list
 
       - name: Running End-to-End tests for Playground application
-        run: pnpm start-server-and-test 'pnpm playground:start:prod:local' http-get://127.0.0.1:3001 'pnpm percy exec -- pnpm test:e2e:playground'
+        run: |
+          pnpm start-server-and-test 'pnpm playground:start:prod:local' http-get://127.0.0.1:3001 'pnpm percy exec -- pnpm test:e2e:playground -- pnpm cypress:coverage'
```

Comment:
I don't think we need to add the `pnpm cypress:coverage` part here at the end of the command. The only thing that it does is to run the e2e tests with the `CYPRESS_COVERAGE` env set to 'true' but the `pnpm test:e2e:playground is already running the e2e tests and right below we're already passing that env variable to the command executed.